datafaker/generators/__init__.py

```diff
@@ -0,0 +1,51 @@
+"""Generators write generator function definitions and queries into config.yaml."""
+
+from functools import lru_cache
+
+from datafaker.generators.base import (
+    ConstantGeneratorFactory,
+    GeneratorFactory,
+    MultiGeneratorFactory,
+)
+from datafaker.generators.choice import ChoiceGeneratorFactory
+from datafaker.generators.continuous import (
+    ContinuousDistributionGeneratorFactory,
+    ContinuousLogDistributionGeneratorFactory,
+    MultivariateLogNormalGeneratorFactory,
+    MultivariateNormalGeneratorFactory,
+)
+from datafaker.generators.mimesis import (
+    MimesisDateGeneratorFactory,
+    MimesisDateTimeGeneratorFactory,
+    MimesisFloatGeneratorFactory,
+    MimesisIntegerGeneratorFactory,
+    MimesisStringGeneratorFactory,
+    MimesisTimeGeneratorFactory,
+)
+from datafaker.generators.partitioned import (
+    NullPartitionedLogNormalGeneratorFactory,
+    NullPartitionedNormalGeneratorFactory,
+)
+
+
+@lru_cache(1)
+def everything_factory() -> GeneratorFactory:
```

Comment:
ooh nice, is there a reason that this is better than making an object here, that's imported? I presume you may want to edit the output?